lib/StatusIndicator/index.js

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const StatusIndicator = props =>
-  (<span className="status-indicator">
-    <span
-      className="status-indicator__circle"
-      style={{ backgroundColor: props.color }}
-    />
-    <span className="status-indicator__label">{ props.label }</span>
-    <span className="status-indicator__children">{ props.children }</span>
-  </span>);
+const StatusIndicator = (props) => {
+  const classNames = ['status-indicator'];
+  let description;
+  if (props.completed) {
+    classNames.push('status-indicator--completed');
+  }
+  if (props.current) {
+    classNames.push('status-indicator--current');
+    description = <p className="status-indicator__description">{ props.description }</p>;
```

Comment:
Conditionals in JSX can be handled as such;

```
{ props.current &&
  <p className="status-indicator__description">{ props.description }</p>
}
```

This can then be moved down to where `{ description }` is. With the feedback around `classNames` I think this approach would be nicer 👍